src/parser/hunter/beastmastery/modules/spells/BarbedShot.tsx

```diff
@@ -27,30 +33,40 @@ class BarbedShot extends Analyzer {
     spellUsable: SpellUsable,
   };
 
-  damage = 0;
-  barbedShotStacks = [];
-  lastBarbedShotStack = 0;
+  barbedShotStacks: any[][];
```

Comment:
Try to avoid using any, as it provides no type information and no safety. It's an escape hatch.

It's sometimes not worth converting complex values in old code though.